test/functional/eval/system_spec.lua

```diff
@@ -255,7 +255,7 @@ describe('system()', function()
       eq(2, eval("1+1"))  -- Still alive?
```

Comment:
off topic, just noticed the test on lin 250 should be:

    feed_command('call system("cat - &", "input")')